src/helpers/form.js

```diff
@@ -49,10 +49,12 @@ export function getFieldValue(field) {
 }
 
 export function getParamsFromFormState(state, list, append = []) {
-    const params = list.map(k => `${k}=${encodeURIComponent(state[k])}`)
+    const params = list.map(k =>
```

Comment:
Maybe this is because one letter variables are also used elsewhere in the app, but just for readability a slightly more elaborate name is nice I think. Make it just that much easier to skim the code if the name is descriptive (unless it clutters it unnecessarily).

Reply:
Same with the `v` vars below

Reply:
Replaced, please review this one